code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -5492,3 +5492,13 @@
 		mix_phrase = "The mixture emits a sudden whine of static and forms into swirling, many faceted shapes that hurt to look at."
 		result_amount = 2
 		mix_sound = 'sound/effects/radio_sweep1.ogg'
+
+	ehsjuice
+		name = "Juice That Makes Your Head Explode"
+		id = "ehsjuice"
+		result = "ehsjuice"
+		required_reagents = list("blackpowder" = 1, "hard_punch" = 1, "juice_raspberry" = 1, "prions" = 1, "sonicpowder" = 1, "water_holy" = 1)
```

Comment:
Could proobably remove at least one of these, since each one other than sonic powder is a Process to get.